packages/wdio-appium-service/src/launcher.js

```diff
@@ -48,10 +49,15 @@ export class AppiumLauncher {
             }
         })
 
+        // only capture first error to print it in case Appium failed to start.
+        process.stderr.once('data', err => { error = err })
+
         process.once('exit', (exitCode) => {
             let errorMessage = `Appium exited before timeout (exit code: ${exitCode})`
             if (exitCode == 2) {
-                errorMessage += ' - Check that you don\'t already have a running Appium service.'
+                errorMessage += '\n' + (error || 'Check that you don\'t already have a running Appium service.')
+                // eslint-disable-next-line no-console
+                console.error(errorMessage)
```

Comment:
```suggestion
                log.error(errorMessage)
```

Reply:
I'm not sure why, but if I replace console.error with log.error I can't see it anywhere. 
Maybe there is some race condition or we are not waiting for log to be flushed. 

I'll check

Reply:
changed it to `log.error`.
log level was set to `warn` in my project and for some reason because of that error messages were not showing up. 
Changed it to `info` fixes the problem, have no idea why is that happening, have to investigate separately.